src/node/transaction.h

```diff
@@ -60,10 +60,11 @@ static const CAmount DEFAULT_MAX_BURN_AMOUNT{0};
  * @param[in]  block_index     The block to read from disk, or nullptr
  * @param[in]  mempool         If provided, check mempool for tx
  * @param[in]  hash            The txid
+ * @param[in]  blockman        Used to access and read blocks from disk
  * @param[out] hashBlock       The block hash, if the tx was found via -txindex or block_index
```

Comment:
Just add the comment instead of changing the order of the params.

```suggestion
 * @param[out] hashBlock       The block hash, if the tx was found via -txindex or block_index
 * @param[in]  blockman        Used to access and read blocks from disk
```

Reply:
Done. Thanks for the suggestion